### PR TITLE
Add CentOS errata tool

### PR DESCRIPTION
**Pull request by Loquacity**
https://github.com/SUSE/spacewalk/issues/10198

### CODE REVIEW

File: modules/client-configuration/pages/clients-centos.adoc [hunk 1]
```diff
@@ -123,3 +123,69 @@ endif::[]
...
+touch /usr/local/bin/cent-errata.sh
+----
+. Edit the new file to include this script, editing the repository details as required:
++
+----
+#!/bin/bash
+mkdir -p /usr/local/centos
+cd /usr/local/centos
+rm *.xml
+wget -c http://cefs.steve-meier.de/errata.latest.xml
```
Comment (keichwa) [inline modules/client-configuration/pages/clients-centos.adoc]: I think we should explain what http://cefs.steve-meier.de is.

File: modules/client-configuration/pages/clients-centos.adoc [hunk 2]
```diff
@@ -123,3 +123,69 @@ endif::[]
...
+--include-channels=centos7-x86_64-updates,centos7-x86_64,centos7-x86_64-extras \
+--publish --rhsa-oval /usr/local/centos/com.redhat.rhsa-RHEL7.xml
+----
+. Set up a cron job to run the script daily:
++
+----
+ln -s /usr/local/bin/cent-errata.sh /etc/cron.daily
+----
+
+For more information on this tool, see https://cefs.steve-meier.de/
```
Comment (keichwa) [inline modules/client-configuration/pages/clients-centos.adoc]:
```suggestion
For more information on this tool, see https://cefs.steve-meier.de/.
```

**Comment by paususe** [comment]
I'm not so sure we want to include this much detail in our docs, especially in the SUSE Manager docs.

I would rather say something more generic, such as "
When you update {centos} clients, the packages do not include metadata about the updates. You can use a third-party errata service to obtain this information, e. g. https://cefs.steve-meier.de/" and then link to wiki.microfocus.com for the full detail (or even move that page to uyuni-project.org or https://github.com/uyuni-project/uyuni/wiki).

@joesusecom @BLandorff What do you think?

**Comment by mcalmer** [comment]
Better not make use of wiki.microfocus.com as we do not know how long it will stay and if it gets migrated at all. Uyuni Wiki might be a good option. But if we want to support CentOS officially with 4.1 we might add this to the official docs.

We could also think about adding the missing package to our repository. Then nobody would need to enable development tools product on SUSE Manager.

**Comment by Loquacity** [comment]
Going to leave this as-is until we have a clear direction from PM.

**Comment by Loquacity** [comment]
@paususe Any updates here? If we don't get something soon, I'll close this PR and we can re-address in the new year.

**Comment by paususe** [comment]
> @paususe Any updates here? If we don't get something soon, I'll close this PR and we can re-address in the new year.

Let's include it in the docs

File: modules/client-configuration/pages/clients-centos.adoc [hunk 3]
```diff
@@ -123,3 +123,65 @@ endif::[]
...
+mkdir -p /usr/local/centos
+cd /usr/local/centos
+rm *.xml
+wget -c http://cefs.steve-meier.de/errata.latest.xml
+#wget -c https://www.redhat.com/security/data/oval/com.redhat.rhsa-all.xml
+wget -c https://www.redhat.com/security/data/oval/com.redhat.rhsa-RHEL7.xml
+wget -c http://cefs.steve-meier.de/errata-import.tar
+tar xvf errata-import.tar
+chmod +x /usr/local/centos/errata-import.pl
+export SPACEWALK_USER='admin';export SPACEWALK_PASS='<password>'
```
Comment (mcalmer) [inline modules/client-configuration/pages/clients-centos.adoc]: 'admin' needs to be replaced with the real name as well like the password.
Not sure if we should write `<user>` .

Comment (keichwa) [inline modules/client-configuration/pages/clients-centos.adoc]:
```suggestion
export SPACEWALK_USER='<adminname>';export SPACEWALK_PASS='<password>'
```

File: modules/client-configuration/pages/clients-centos.adoc [hunk 4]
```diff
@@ -123,3 +123,65 @@ endif::[]
...
+/usr/local/centos/errata-import.pl --server <servername> \
```
Comment (keichwa) [inline modules/client-configuration/pages/clients-centos.adoc]:
```suggestion
/usr/local/centos/errata-import.pl --server '<servername>" \
```

**Pull request merged**